src/app/query/bitcoin/address/address.hooks.ts

```diff
@@ -0,0 +1,21 @@
+// @ts-nocheck
+import { sumByKey } from '@app/common/utils';
+import { BTC_DECIMALS } from '@shared/constants';
+
+import { useGetTransactionsByAddressQuery } from './transactions-by-address.query';
+import { useGetUtxosByAddressQuery } from './utxos-by-address.query';
+
+// TODO: Export when needed and remove ts-nocheck
+function useGetTransactionsByAddress(address: string) {
+  return useGetTransactionsByAddressQuery(address).data;
+}
+
+function useGetUtxosByAddress(address: string) {
+  return useGetUtxosByAddressQuery(address).data;
+}
+
+function useGetBalanceByAddress(address: string) {
+  const utxos = useGetUtxosByAddressQuery(address).data;
+  if (!utxos) return;
+  return sumByKey(utxos, 'value').shiftedBy(-BTC_DECIMALS);
```

Comment:
As in `simplerSumMethodThatTakesArray(utxos.map(utxo => utxo.value))`

Not mandatory to change, but worth considering which is simpler.